packages/tasks/src/tasks/text-generation/spec/output.json

```diff
@@ -1,165 +1,197 @@
 {
 	"$id": "/inference/schemas/text-generation/output.json",
 	"$schema": "http://json-schema.org/draft-06/schema#",
-	"description": "Outputs for Text Generation inference",
+	"description": "Text Generation Output",
 	"title": "TextGenerationOutput",
 	"type": "object",
+	"required": ["generated_text"],
 	"properties": {
+		"details": {
+			"allOf": [
+				{
+					"$ref": "#/$defs/TextGenerationOutputDetails"
+				}
+			],
+			"nullable": true
+		},
 		"generated_text": {
 			"type": "string",
-			"description": "The generated text"
-		},
-		"details": {
-			"$ref": "#/$defs/Details",
-			"description": "When enabled, details about the generation"
+			"example": "test"
 		}
 	},
-	"required": ["generated_text"],
 	"$defs": {
-		"FinishReason": {
-			"type": "string",
-			"title": "TextGenerationFinishReason",
-			"description": "The reason why the generation was stopped.",
-			"oneOf": [
-				{ "const": "length", "description": "length: The generated sequence reached the maximum allowed length" },
-				{ "const": "eos_token", "description": "eos_token: The model generated an end-of-sentence (EOS) token" },
-				{
-					"const": "stop_sequence",
-					"description": "stop_sequence: One of the sequence in stop_sequences was generated"
-				}
-			]
-		},
-		"PrefillToken": {
-			"title": "TextGenerationPrefillToken",
+		"TextGenerationOutput": {
```

Comment:
Same here - it seems this type is decalred twice (once at root level & once here in `$defs`)

Reply:
Fixed in https://github.com/huggingface/huggingface.js/pull/629/commits/61b9d4e67f948239d3c4832ccabee866e2b67f91. Thanks for reporting :)